AuctionHouse/Overview.md

```diff
@@ -0,0 +1,129 @@
+
+> Example:
+> * Alice lists an asset A for 5 SOL, thus the Auction House creates a **Sell Order** for the asset A.
+> * Bob places a bid of 3 SOL for the asset A. The Auction House creates a **Buy Order** for the asset A.
+> * Alice agrees to the offer, thus enabling the marketplace to "execute the sale" using the `executeSale` instruction.
+
+2. *"Buying" at list price*: This is the case when the bidder places a bid for an already listed asset, at the list amount itself. In such cases, the `executeSale` instruction is executed in the same transaction in which the bid is made (the **Buy Order** is made), and thus the bidder "buys" the asset.
+
+> Example:
+> * Alice lists an asset A for 5 SOL, thus the Auction House creates a **Sell Order** for the asset A.
+> * Bob places a bid of 5 SOL for the asset A, creating a **Buy Order** for the asset A.
+> * This enables the marketplace to place a bid on the asset and execute the sale in the same transaction, in practice allowing Bob to "buy" asset A.
+
+3. *"Selling" at bid price*: This is the case when a user interested in an unlisted asset places a bid on it. Once this happens, if the asset owner, now interested in selling the asset, lists the asset for the bid amount, the `executeSale` instruction is executed in the same transaction in which the listing is done (the **Sell Order** is made), and thus the lister "sells" the asset.
+
+> Example:
+> * Bob places a bid of 5 SOL for an unlisted asset A. 
+> * Alice notices the bid and lists the asset A for 5 SOL, creating a **Sell Order** for the asset A. 
+> * This enables the marketplace to list the asset and execute the sale in the same transaction, in practice allowing Alice to "sell" asset A.
+    
+## Auctioning Fungible Assets
+So far, we've talked about exchanging assets using an Auction House account, but we've not dug into what type of assets can be exchanged that way. The most popular assets that can be listed in an Auction House are [Non-Fungible Tokens (NFTs)](/resources/definitions#non-fungible-tokens).
+
+However, these are not the only assets that can benefit from the Auction House program. In fact, an asset can be any SPL Token so long as it has a Metadata account attached to its Mint account. If you'd like to know more about SPL Token and Metadata accounts, you can [read more about this in the overview of our Token Metadata program](/programs/token-metadata/overview).
+    
+## Buying asset using a custom SPL Token
+In all the examples showcased above, we use SOL as the exchange currency to discuss how the Auction House program works. But SOL is not the only currency that can be configured to be used for exchanging assets and the Auction House program allows the marketplaces to configure any SPL-token to act as the currency for exchanging assets on their platform.
```

Comment:
This is very good!

The sentences are a bit too long though. How about...

```suggestion
In the examples showcased above, we've used SOL as the exchange currency to discuss how the Auction House program works. But SOL is not the only currency that can be configured for exchanging assets. The Auction House program allows marketplaces to configure any SPL-token to act as their currency.
```

I think in general, your sentences are very good but good have much more impact by simplifying them slightly. Remove a few redundant words so readers get to the point quicker. Insert smaller sentences here and there to control the flow. Allow them to breathe. See what I did there. 😅

Reply:
Completely understand your point, that's a great suggestion Loris. I can notice I've used words that are causing pleonasm in the sentences. Will make sure to cut these off :)